app/models/foreman_virt_who_configure/output_generator.rb

```diff
@@ -197,6 +198,10 @@ def config_file_path
     end
 
     def sysconfig_file_path
```

Comment:
Does it make sense to update this method name since `sysconfig` is kind of a misnomer?

Reply:
@jturel I was going to use `config_file_path` but that is already taken by this guy:

https://github.com/theforeman/foreman_virt_who_configure/blob/4f6f8a2aabb694bd87fbd0db3255ddc175e622f9/app/models/foreman_virt_who_configure/output_generator.rb#L196

What do you think of renaming `config_file_path` to `user_config_file_path` and my change to `config_file_path`?

Reply:
Or maybe s/sysconfig_file_path/default_config_path ?

Reply:
Works for me, updated.